src/components/Map/Map.js

```diff
@@ -128,10 +128,21 @@ class Map extends PureComponent {
 
       // don't continue if there's a popup that was opened by click
       if (!isClickInfoOpen) {
-        infos = infos.filter(
-          ({ layer }) =>
-            layer.get('showPopupOnHover') && layer.get('popupComponent'),
-        );
+        infos = infos
+          .filter(
+            ({ layer }) =>
+              layer.get('showPopupOnHover') && layer.get('popupComponent'),
+          )
+          .map(i => {
```

Comment:
call it `info`  we don' t understand what is `i`

Reply:
Done